utils/update_checkout/update-checkout-config.json

```diff
@@ -351,25 +351,26 @@
        "tensorflow": {
             "aliases": ["tensorflow"],
             "repos": {
-                "llvm-project": "a2905f3b8df7784e5c455390c6c0b0f7564e00e4",
+                "llvm-project": "2e4f104624f9cdc7c8cb471a393f496b48922c0b",
                 "swift": "tensorflow",
-                "cmark": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
-                "llbuild": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
+                "cmark": "swift-DEVELOPMENT-SNAPSHOT-2020-05-18-a",
+                "llbuild": "swift-DEVELOPMENT-SNAPSHOT-2020-05-18-a",
                 "swift-tools-support-core": "0.0.1",
-                "swiftpm": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
+                "swiftpm": "swift-DEVELOPMENT-SNAPSHOT-2020-05-20-a",
```

Comment:
@ematejska NB: I needed to adjust the swiftpm dependency to pick up https://github.com/apple/swift-package-manager/commit/3f55ea6d555ee8a73013eea44a50567fb69e9f41.

Reply:
Ok, got it.